Gemfile.lock

```diff
@@ -11,30 +11,32 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (5.2.0)
+    activesupport (6.0.3.2)
```

Comment:
i think we may have a problem with upgrading to 6, isn't that rails 6 specific?

Reply:
fixed